Code/MinimalLib/demo/demo.html

```diff
@@ -16,17 +16,45 @@
     }
   </style>
 </head>
-<script src="./RDKit_minimal.js"></script>
+<script id="minimalLib"></script>
```

Comment:
These changes are only necessary if we want to have one demo.html file which supports both the WASM version and legacy JS version, right?

If so: I would suggest we not do this at all. The demo.html can remain simple and cover the 99% case.

Reply:
Yes, the dynamic selection of the library at runtime based on whether the browser supports WebAssembly or not is only needed to have a single file that works for both use cases. I see your point, but having two HTML files means that each time we update the main file we also need to update the other file, which is quite inconvenient. And I would say it is not a bad thing that it is quite simple and convenient to do so. Dynamically  Injecting a JS library into an HTML page is a pretty standard operation, and incidentally it is the same thing we to support interactive rendering in Jupyter.
It is your call, but I would leave this as is. We can make make the WebAssembly check less paranoid (you don't _actually_ need to run some code to prove WebAssembly is supported) if you prefer. But I would really vote against duplicating the HTML file.

Reply:
@ptosco how about just documenting that the demo.html only works with the webassembly version and not have a version for the pure JS one since that's for a very specialized situation?

Reply:
I realized the highlighting was broken in legacy browsers by loading `demo.html`.
Let me see if I can find a prettier way to get `demo.html` to work for both.

Reply:
Hello, sorry for the delay. Just for the record, the "legacy" (or so they are called in rdki-tjs) demos that are linked to the rdkitjs.com site are these files https://github.com/rdkit/rdkit-js/tree/master/docs , and not the ones in the rdkit main repo anymore (those currently edited in this PR).